src/TestFramework/Extension.UWP/Extension.UWP.csproj

```diff
@@ -50,15 +37,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="$(MicrosoftNETCoreUniversalWindowsPlatformVersion)" />
-    <PackageReference Include="System.Net.NameResolution" Version="$(SystemNetNameResolutionVersion)" />
```

Comment:
Some of these dependencies are also declared in nuget package. Shall we remove them too?